src/cljc/orcpub/dnd/e5/classes.cljc

```diff
@@ -2601,7 +2603,16 @@
                              melee-weapons-xform
                              weapons)})]}))
 
-(defn pact-boon-options [spell-lists spells-map]
+(defn pact-boon-options [plugin-boons spell-lists spells-map]
+ (concat
+   (map
```

Comment:
you can use `(mapcat ..)`  instead of `(concat (map ...))`

Reply:
Apparently this won't work. Probably because `mapcat` replaces `(apply concat (map ...)`, but not `(concat (map ...)`

But wait @marloso2 , is `(concat (map... ` what you want? `concat` takes multiple collections, but `map` only returns one. I just tested in a REPL, and `(concat (map ...` is no different than plain `map`.

Reply:
Iirc, that bit of code is where the plugins are combined with the predefined options. The `map` lists out the plugins to then be `concat`ed with the ones built into the code.

Reply:
Which, now that I think about it, explains why `mapcat` didn't work for this case... and why both commands are necessary

Reply:
Ahhh, it's concatting with the vector of pacts afterwards, my bad. You're right, then.